types/types.go

```diff
@@ -84,6 +84,7 @@ const (
 	LonghornLabelBackupVolume             = "backup-volume"
 	LonghornLabelRecurringJob             = "job"
 	LonghornLabelRecurringJobGroup        = "job-group"
+	LonghornLabelCRDVersion               = "crd-version"
```

Comment:
It's better to call it (Longhorn) API version rather than CRD version...

Reply:
longhorn.io/v1beta1 and longhorn.io/v1beta2 are extended APIs of Longhorn.
For the CRD version, it's apiextensions.k8s.io/v1beta1 and apiextensions.k8s.io/v1 from my point of view.
So, personally, I think Shou's comment is fair.

Reply:
Understood! Thanks for the explanation.